nwb.config.js

```diff
@@ -1,19 +1,53 @@
-const browsers = require('./test/browsers');
+const babel = require('babel-core');
 const externals = require('webpack-node-externals');
-const [ cmd ] = require('yargs').argv._;
+const fs = require('fs');
+const glob = require('glob');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const rimraf = require('rimraf');
+const yargs = require('yargs');
+
+const browsers = require('./test/browsers');
+
+const [ cmd ] = yargs.argv._;
+const isBuild = cmd === 'build';
+const babelConfig = {
+  plugins: ['transform-react-jsx']
+};
+
+if (isBuild) {
+  const files = glob.sync('src/**/*.js');
+  rimraf.sync('./es');
+  files.forEach(f => {
```

Comment:
> I'm just not used to for such kind of complexity to just emit raw es6 to some other folder ( in TS this is just one liner from cli :-| )

Let's set the TypeScript stuff aside, please. I think I can probably work out a simple command for it, I was just sketching off the top of my head.